main.py

```
@@ -407,6 +407,96 @@ def main():
                 print("✅ Switched to phi3:mini successfully")
             else:
                 print("❌ Failed to switch to phi3:mini")
+
+        # OCRS Wrapper options
+        elif command == "ocr-img":
+            # Usage: python main.py ocr-img <image_file> [--img]
+            import subprocess
+            args = sys.argv[2:]
+            cmd = ["ocrs_wrapper.py"] + args
```

Ruff (0.13.1)

Consider `["ocrs_wrapper.py", *args]` instead of concatenation

Replace with `["ocrs_wrapper.py", *args]`

(RUF005)